dev-scheme/guile/guile-2.2.3.recipe

```diff
@@ -0,0 +1,109 @@
+SUMMARY="GNU Ubiquitous Intelligent Language for Extensions"
+DESCRIPTION="Guile is a library designed to help programmers create flexible \
+applications. Using Guile in an application allows programmers to write \
+plug-ins, or modules (there are many names, but the concept is essentially \
+the same) and users to use them to have an application fit their needs."
+HOMEPAGE="https://www.gnu.org/software/guile/"
+COPYRIGHT="1993-2018 Aubrey Jaffer, George Carrette, et al."
+LICENSE="GNU LGPL v3"
+REVISION="1"
+SOURCE_URI="https://ftp.gnu.org/gnu/guile/guile-2.2.3.tar.gz"
```

Comment:
Suggestion:
```diff
-SOURCE_URI="https://ftp.gnu.org/gnu/guile/guile-2.2.3.tar.gz"
+SOURCE_URI="https://ftpmirror.gnu.org/guile/guile-$portVersion.tar.gz
+	https://ftp.gnu.org/gnu/guile/guile-$portVersion.tar.gz"
```
(Notice that the "/gnu/" is only required for the second URI. It is not needed for the first one.)